src/backend/gporca/libgpopt/include/gpopt/xforms/CXformFactory.h

```diff
@@ -111,10 +111,10 @@ class CXformFactory
 	}
 
 	// initialize global factory instance
-	static GPOS_RESULT Init();
+	static void Init();
 
 	// destroy global factory instance
-	void Shutdown();
+	static void Shutdown();
```

Comment:
This change does not depend on replacing of error codes checking with interception of exceptions. So adding "static" should be in another commit

Reply:
No! See using CXformFactory::Shutdown.

Reply:
done